### PR TITLE
Feature/clear tasks

### PR DESCRIPTION
**Pull request by starkipraggy**
This closes issue #33 
- [x] ClearTask Function
- [x] UnitTests

### CODE REVIEW

**Comment by bsmmoon** [comment]
:+1:

File: DoLah/Models/Calendar.cpp [hunk 1]
```diff
@@ -1,35 +1,41 @@
...
 
-	void Calendar::addTask(AbstractTask* task) {
-		//TODO
+	std::vector<AbstractTask*> Calendar::getTaskList() {
+		return taskList;
 	}
 
-	void deleteTask(int id) {
-		//TODO
+	void Calendar::addTask(AbstractTask* task) {
```
Comment (9gix) [inline DoLah/Models/Calendar.cpp]: Can we have all argument passed by const reference instead?
Use pointer as argument only when no other choice.

File: DoLah/Models/Calendar.cpp [hunk 2]
```diff
@@ -1,35 +1,41 @@
...
 	}
 
-	void updateTask(AbstractTask*) {
-		//TODO
+	void Calendar::deleteTask(int id) {
+		
 	}
 
-	AbstractTask* getTask(int id) {
+	void Calendar::updateTask(AbstractTask*) {
```
Comment (9gix) [inline DoLah/Models/Calendar.cpp]: unnamed argument.

**Comment by 9gix** [comment]
Please update your code as commented above. or discuss if you don't think that it is necessary.

**Comment by 9gix** [comment]
Merged into #75

**Pull request closed without merging**